Remove mobile conditions

kevinsimper opened the pull request:
Remove a lot of mobile behavior, which as smartphones gets bigger screens it not wanted behavior.

Removing scroll on viewport is not something you want as it also happends on a iPad for an example.
The window.scrollTo does not work on iOS7 anymore, so that is also useless now, also because it will not scroll if the viewport height is equal to the document height.
Setting that the touchmove on is ignored on the Crafty element and not on the whole page, so that the webpage is still functional.

There should properly be something like this, but is should be optional and thought through, which this is not.

starwed commented:
I'm not sure if we want to completely strip this away.

Having an option to disable it is one thing, or even making it off by default.  But in a lot of games you _would_ want it to be fullscreen on mobile.

kevinsimper commented:
Yes, and as mentioned there should be something like this, but this has unintended effects like you can't scroll in the tests when you load /test/index.html.
Also on craftyjs.com if you are on an iPad and there is an example you can not scroll.

There should be something like "do fullscreen on mobile devices" option, when you init the game.

There is also some in the forum that complain about it that Crafty does not respect the values in the init function.

For an example (and they are just the ones who took the time to complain)
#663 
https://groups.google.com/forum/#!searchin/craftyjs/mobile/craftyjs/HtFTCHFfAko/-rLp-Tdbc5kJ
https://groups.google.com/forum/#!searchin/craftyjs/mobile/craftyjs/4tGr_2ou964/hlorng3W6zkJ
https://groups.google.com/forum/#!searchin/craftyjs/mobile/craftyjs/NXu4F8b_-Fg/cguuyMvjq7QJ 
https://groups.google.com/forum/#!searchin/craftyjs/mobile/craftyjs/pwvCiJetMN4/dbUaoBNbqVUJ

kevinsimper committed:
Remove unwanted mobile behavior

Removing scroll on viewport is not something you want as it also happens on a iPad for an example.
The window.scrollTo does not work on iOS7 anymore, so that is also useless now, also because it will not scroll if the viewport height is equal to the document height.
Setting that the touchmove on is ignored on the Crafty element and not on the whole page, so that the webpage is still functional.

kevinsimper commented:
This is an important fix and relative small change, because if you want to go fullscreen on mobile you can just do `Crafty.init()`, just like you would on the computer.

These changes are easy to add yourself, but hard to remove from Crafty when used.
